Easydict.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -298,5 +308,5 @@
       }
     }
   ],
-  "version" : 2
+  "version" : 3
```

Comment:
Try to revert this line and build again. I believe this change caused the build issue.

Reply:
It's strange, I didn't manually change this value, it was modified by Xcode itself after adding OpenAI, and there were no errors when compiling locally.

Let's try to revert it.

Reply:
ok, it did work out.